diesel_tests/Cargo.toml

```diff
@@ -17,6 +17,7 @@ chrono = { version = "0.4" }
 diesel = { path = "../diesel", default-features = false, features = ["quickcheck", "chrono", "uuidv07", "serde_json", "network-address", "numeric", "with-deprecated"] }
 diesel_migrations = { version = "1.4.0" }
 diesel_infer_schema = { version = "1.4.0" }
+diesel_derives = { version = "1.4" }
```

Comment:
not exactly `1.4.0` ?

Reply:
No, I did not change these too, it should be in another issue.

Reply:
I'm a little lost on the context of the reply.

I was just trying to say that 

`diesel_derives = { version = "1.4" }`
should be
`diesel_derives = { version = "1.4.0" }`

like the others around it
```
diesel_migrations = { version = "1.4.0" }
diesel_infer_schema = { version = "1.4.0" }
```

I should really be using the change suggestion feature on github.

Reply:
Sorry, @martell I just noticed that my context was also wrong, I thought I was in another repository and another pull request, it also have a similar name.